src/main/kotlin/parts/wisdom/arcraider/visualization/Grid.kt

```diff
@@ -1,28 +1,43 @@
 package parts.wisdom.arcraider.visualization
 
 import parts.wisdom.arcraider.SerializedGrid
+import java.awt.Color
+import java.lang.IllegalStateException
 
-data class Coord( val x : Int, val y : Int)
+data class Square(val x : Int, val y : Int, val color: Int)
```

Comment:
Hmm... I'm not a big fan of using "Square" for this, because we are already talking about drawing lines and stuff. What about "Cell"?